miniapp/src/main/java/com/rakuten/tech/mobile/miniapp/storage/MiniAppStorage.kt

```diff
@@ -19,13 +19,14 @@ internal class MiniAppStorage(
     ) {
         val filePath = getFilePath(source)
         val fileName = getFileName(source)
-        fileWriter.write(inputStream, getParentPath(basePath, filePath), fileName)
+        fileWriter.write(inputStream, getAbsoluteWritePath(basePath, filePath, fileName))
     }
 
-    fun getParentPath(
+    fun getAbsoluteWritePath(
         basePath: String,
-        filePath: String
-    ) = "$basePath$filePath"
+        filePath: String,
+        fileName: String
+    ) = "${basePath.removeSuffix("/")}$filePath$fileName"
```

Comment:
This works for now, but in the future I think we will need to make sure that the `/` isn't removed if it needs to be there - i.e. if `filePath` doesn't start with a `/`.